examples/with-wallet-stamper/src/components/turnkey-provider.tsx

```diff
@@ -101,13 +98,7 @@ export const TurnkeyProvider: React.FC<TurnkeyProviderProps> = ({
     chainType: ChainType = ChainType.SOLANA
   ) {
     setAuthenticating(true);
-    let publicKey = null;
-    if (chainType === ChainType.SOLANA) {
-      const solanaWallet = wallet as SolanaWalletInterface;
-      publicKey = solanaWallet.recoverPublicKey();
-    } else if (chainType === ChainType.EVM) {
-      const evmWallet = wallet as EvmWalletInterface;
-    }
+    const publicKey = await wallet?.getPublicKey();
```

Comment:
in the EVM case, this will require another passkey tap correct?

Reply:
yea unfortunately, I think we could brain storm about maybe saving the key in local storage or in memory?

Reply:
@andrewkmin oh that is interesting, although it looks like is restricted to Local accounts where the privatekey is stored on the user's machine

Reply:
yep, not sure if it's compatible with external signers but interesting nonetheless